test/unit/web_extension_test.rb

```diff
@@ -12,6 +12,7 @@ def app
   before do
     env 'rack.session', { csrf: TOKEN }
     env 'HTTP_X_CSRF_TOKEN', TOKEN
+    Sidekiq::Cron.reset!
```

Comment:
Maybe we can remove it from here? 🤔 I'm almost sure in this test we don't modify the settings.

Reply:
Sure